internal/controllers/nmc_reconciler_test.go

```diff
@@ -143,6 +158,9 @@ var _ = Describe("NodeModulesConfigReconciler_Reconcile", func() {
 			reflect.TypeOf((*context.Context)(nil)).Elem(),
 		)
 
+		kubeClient.EXPECT().Get(ctx, types.NamespacedName{Name: nmc.Name}, &node).Return(err)
```

Comment:
```suggestion
		kubeClient.EXPECT().Get(ctx, types.NamespacedName{Name: nmc.Name}, &node).Return(nil)
```

If we don't return any error, then let's return `nil` it is clearer then searching what `err` got as its value.

Also, any reason we don't put this line inside the `InOrder` block?

Reply:
fixed, moved it also to the InOrder section